### PR TITLE
📖 Enhance docs

### PR DESCRIPTION
**Pull request by fxierh**
List disabled operators (cloud-credential-operator, cluster-autoscaler-operator, cluster-cloud-controller-manager-operator) in the docs. 

Also incorporate a few minor adjustments.

### CODE REVIEW

**Comment by gbraad** [comment]
The fixes for documentation and makefile should perhaps be split. They are unrelated/independent.

**Comment by fxierh** [comment]
@gbraad Yup, move the part for make targets to #3968.

File: docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ Upgrading the {ocp} version might cause issues that are difficult to reproduce.
 * It uses a single node, which behaves as both a control plane and worker node.
 * It disables the Cluster Monitoring Operator by default.
 This disabled Operator causes the corresponding part of the web console to be non-functional.
+* It does not include the Cloud Credential Operator.
 * The {ocp} cluster runs in a virtual machine known as an __instance__.
```
Comment (anjannath) [inline docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc:15]: there are more cluster operators that are currently disabled maybe that should also be listed, from: https://github.com/crc-org/snc/blob/master/cvo-overrides.yaml

- cluster-autoscaler
- cloud-controller-manager

Comment (fxierh) [reply docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc:15]: @anjannath Yup, done.

**Comment by anjannath** [comment]
LGTM, /cc @themr0c please take a look

File: docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc (hunk 1 above)
Comment (gbraad) [reply docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc:15]: A lot of repetition in tone/structure.

Perhaps best to use a list here:

* It disables the following operators:
  - Cloud Credential Operator
  - Cluster Autoscaler Operator
  - Cluster Cloud Controller Manager Operator
  - Cluster Monitoring Operator

@themr0c WDYT?

Comment (fxierh) [reply docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc:15]: @gbraad I find it somewhat challenging to include the Cluster Monitoring Operator in the same list as the other three. Unlike the others, it is only disabled 'by default' (can be enabled day 1 if needed) and its disablement results in the corresponding section of the web console becoming non-functional. 

How about this:
```adoc
* The Cluster Monitoring Operator is disabled by default. This results in the corresponding part of the web console to be non-functional.
* The following operators are always disabled:
** Cloud Credential Operator.
** Cluster Autoscaler Operator.
** Cluster Cloud Controller Manager Operator.
```

File: docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc [hunk 2]
```diff
@@ -8,6 +8,9 @@ Upgrading the {ocp} version might cause issues that are difficult to reproduce.
 * It uses a single node, which behaves as both a control plane and worker node.
 * It disables the Cluster Monitoring Operator by default.
 This disabled Operator causes the corresponding part of the web console to be non-functional.
+* It disables the Cloud Credential Operator.
+* It disables the Cluster Autoscaler Operator.
+* It disables the Cluster Cloud Controller Manager Operator.
```
Comment (themr0c) [inline docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc]: This sounds fine:

```suggestion
* It disables the following operators:
** Cloud Credential Operator.
** Cluster Autoscaler Operator.
** Cloud Controller Manager Operator.
```

(Keep avoiding passive voice).

**Comment by fxierh** [comment]
Comments addressed.

**Review by themr0c — APPROVED** [review]

**Pull request merged**